readthedocs/projects/models.py

```diff
@@ -558,6 +560,27 @@ def conf_dir(self, version=LATEST):
         if conf_file:
             return os.path.dirname(conf_file)
 
+    @property
+    def is_abandoned(self):
+        """Is project abandoned."""
+        if self.has_good_build:
+            latest_build = self.get_latest_build()
+            if latest_build:
+                latest_build_date = latest_build.date
+                today = datetime.today()
+                diff = today - latest_build_date
+                # Latest build a year ago.
+                if diff > timedelta(days=365):
+                    return True
+                return False
+            return False
+        return True
+
+    @property
+    def is_abandoned_mail_sent(self):
+        """Is abandoned mail sent."""
+        return self.abandoned_mail_sent
```

Comment:
I don't believe this is necessary, you can just use `obj.abandoned_mail_sent` directly.